bin/node/runtime/Cargo.toml

```diff
@@ -45,6 +44,7 @@ frame-support = { version = "2.0.0", default-features = false, path = "../../../
 frame-system = { version = "2.0.0", default-features = false, path = "../../../frame/system" }
 frame-system-benchmarking = { version = "2.0.0", default-features = false, path = "../../../frame/system/benchmarking", optional = true }
 frame-system-rpc-runtime-api = { version = "2.0.0", default-features = false, path = "../../../frame/system/rpc/runtime-api/" }
+pallet-assets = { version = "2.0.0", default-features = false, path = "../../../frame/assets" }
```

Comment:
What is `pallet-assets` responsible for?

Reply:
We don’t need this functionality. 
Polkadot doesn’t use this pallet in [0.8.27](https://github.com/paritytech/polkadot/blob/v0.8.27/runtime/polkadot/Cargo.toml). But it uses it in [older](https://github.com/paritytech/polkadot/commit/f143113b16b448aa56caa70bc6990cee19a4cc69) versions in Cross Consensus Message.
[Link to our testing log](https://www.notion.so/cere/Testing-logs-1f7d5107060a432398bad8b6a15225f0#321a8dd8b7944fcdadff61d3e05e0c10)